src/testing/h_refinement_study_isentropic_vortex.cpp

```diff
@@ -141,8 +172,12 @@ int HRefinementStudyIsentropicVortex<dim, nstate>::run_test() const
         pcout << "---------------------------------------------" << std::endl;
```

Comment:
At the beginning of the `c` loop, there should be output like the above lines to tell the user what the current `c` value is.